server/aws-lsp-codewhisperer/src/language-server/codeWhispererService.ts

```diff
@@ -49,6 +49,7 @@ export abstract class CodeWhispererServiceBase {
     protected readonly codeWhispererRegion = 'us-east-1'
     protected readonly codeWhispererEndpoint = 'https://codewhisperer.us-east-1.amazonaws.com/'
     public shareCodeWhispererContentWithAWS = false
+    public customizationArn: string = ''
```

Comment:
nit: better to use `undefined` or `null` as negative fallback, instead of empty string. JS is weird ofc with treating empty string as false, but we can be explicit :)